config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presets.yaml

```diff
@@ -38,3 +38,30 @@ presets:
   - mountPath: /var/run/secrets/azure-token/serviceaccount
     name: azure-token
     readOnly: true
+        key: serviceAccountSigningKey
+- labels:
+    preset-azure-community-wi: "true"
```

Comment:
I think we can omit the "-wi" suffix here, wdyt @jsturtevant?

Reply:
works for me, I don't have any strong opinions on the naming